packages/openneuro-app/src/scripts/search/search-params-ctx.tsx

```diff
@@ -20,6 +20,10 @@ export const SearchParamsProvider: React.FC<SearchParamsProviderProps> = ({
     const query = searchQuery.get("query")
     if (query) {
       searchParams = JSON.parse(query)
+    } else {
+      Sentry.captureException(
+        "No query found in URL. Using initialSearchParams.",
+      )
```

Comment:
This is probably common enough to not qualify as an exception. A breadcrumb tracking it might be useful though.
```suggestion
      Sentry.addBreadcrumb({
        category: "routing",
        message: "No query found in URL. Using initialSearchParams.",
        level: "info"
      })
```